src/main/plugin/iso19135/layout/layout-custom-fields.xsl

```diff
@@ -8,9 +8,29 @@
                 xmlns:gn-fn-core="http://geonetwork-opensource.org/xsl/functions/core"
                 xmlns:gn-fn-metadata="http://geonetwork-opensource.org/xsl/functions/metadata"
                 xmlns:gn-fn-iso19139="http://geonetwork-opensource.org/xsl/functions/profiles/iso19139"
+                xmlns:xlink="http://www.w3.org/1999/xlink"
                 exclude-result-prefixes="#all">
 
 
+  <!-- Simply show the full content of the list -->
+  <xsl:template name="iso19135-list-full-content">
+    <xsl:apply-templates select="$metadata//grg:containedItem[not(@xlink:href)]"
+        mode="mode-iso19135" />
+  </xsl:template>
+
+  <xsl:template name="iso19135-list-full-content-index">
+    <xsl:variable name="items" select="$metadata//grg:RE_RegisterItem/grg:name/gco:CharacterString"/>
+    <h3>
+        <xsl:value-of select="gn-fn-metadata:getLabel($schema, 'codeListIndexSide', $labels)/label"/>
+    </h3>
+    <ul>
+      <xsl:for-each select="$items">
+        <xsl:variable name="ref" select="../../../gn:element/@ref" />
+        <li><a onclick="document.getElementById('gn-el-{$ref}').scrollIntoView();" ><xsl:value-of select="."/></a></li>
```

Comment:
You could maybe use editor permalink for this https://github.com/geonetwork/core-geonetwork/pull/2591 to avoid JS here ?

Reply:
I have tried that, but the issue is that the permalink reloads the editor page, that is a problem as any editing done is lost when you select the link.

Also seem requires certain html structure, otherwise fails, I tried this:

http://localhost:8080/geonetwork/srv/eng/catalog.edit?debug#/metadata/115?tab=default&scrollTo=gn-el-86

The element id exists, but there's a javascript error in https://github.com/geonetwork/core-geonetwork/blob/master/web-ui/src/main/resources/catalog/components/utility/UtilityService.js#L84

```
Uncaught TypeError: Cannot read property 'top' of undefined
    at eval (eval at scrollTo (UtilityService.js:83), <anonymous>:1:22)
```

Reply:
Ok so forget it I'll have a look if I can use such permalink to link an error in the validation panel and the form

Reply:
I did some changes in the inline validation branch for that https://github.com/geonetwork/core-geonetwork/pull/3298/commits/86da575d6b871718b856ca70931313a91f1b7499 that you can maybe use at some point.